test/runtests.jl

```diff
@@ -257,14 +257,37 @@ end
         @test Matrix(HA.Q) ≈ Matrix(HB.Q)
     end
 end
+@testset "pfaffian.jl" begin
+    for n in [2,4,6]
+        A=rand(1:10,n,n)
+        for i=1:n
+            A[i,i]=0
+            for j=i+1:n
+                A[i,j]=-A[j,i]
+            end
+        end
```

Comment:
```suggestion
        A = skewhermitian!(rand(-10:10,n,n))
```
(and add an specialized `Integer` version if needed?)